app/views/mechanics/show.html.erb

```diff
@@ -0,0 +1,18 @@
+<h1> Mechanic Show Page </h1>
+
+<p>Name: <%= @mechanic.name %></p>
+<p>years_experience: <%= @mechanic.years_experience %></p>
```

Comment:
You have the labels here which is good, so you just need to include them in your specs.